docs/integrations/django.md

```diff
@@ -20,6 +20,9 @@ urlpatterns = [
 ]
 ```
 
+Strawberry core only provides a GraphQL view for Django, Strawberry GraphQL Django provides integration with the models.
```

Comment:
```suggestion
Strawberry only provides a GraphQL view for Django, [Strawberry GraphQL Django](https://github.com/strawberry-graphql/strawberry-graphql-django) provides integration with the models.
```

IMHO a link to strawberry-graphql-django would be useful :)

nitpick, feel free to ignore: we might want to drop "core" from the wording. I know Patrick used it but technically the core library is just called Strawberry (or Strawberry GraphQL), could be confusing to some to suddenly read "core".